examples/scroll/README.md

```diff
@@ -0,0 +1,28 @@
+# Scroll Elements Into View
+
+**Test Code**: [index.js](index.js)  
+
+TestCafe [actions](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/interact-with-the-page.html) have a built-in scrolling mechanism. In rare cases you might need to scroll the page manually. This example features two common ways to do this: a `hover` action and a [ClientFunction](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/obtain-client-side-info.html)
```

Comment:
> This example features

*The example demonstrates* I guess because I feel like a product features something it can offer to the customers (*The new pone features a 120Hz screen*). Here we just show

Reply:
> This example demonstrates two common ways to do this:...